walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,12 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    String n ="";
+    for(int i = 0;i<name.length();i++){
+        if(Character.isLetter(name.charAt(i))){
+            n = n+ name.charAt(i);
+        }
```

Comment:
Really nit-picky but could be a good idea to have more consistent spacing. Like n = n + name.charAt(i) instead of n+, and also in the for loop

Reply:
I should work on having consistent spacing, thank you!

Reply:
:)